version.props

```diff
@@ -4,7 +4,7 @@
     <MinorVersion>0</MinorVersion>
     <PatchVersion>0</PatchVersion>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
-    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview3</PreReleaseVersionLabel>
```

Comment:
Another option is to make this `preview4` immediately and back up to `preview3` in the release branch. Up to you.

Reply:
We don't have a release branch yet for preview3, and I think we're going to hold off on making it for a couple weeks.